org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/thingTranslator.js

```diff
@@ -249,12 +249,16 @@ class Translator {
         return this.postDeviceResource(deviceId, "colourChroma", payload);
     }
 
-    postSubscribe(callbackUrl, verificationRequest) {
-        return winkHub._subscribe(deviceType, deviceId, callbackUrl, verificationRequest);
+    postSubscribe(subscriptionInfo) {
```

Comment:
>postSubscribe [](start = 4, length = 13)

how are we testing this (if we have support for this in the CLI that's one way, in addition we'd also need to update our unit tests for this support) ; Also is this chained all the way up to the hubcontroller?